### PR TITLE
Fix download_url for txt and csvw metadata downloads

### PR DESCRIPTION
**Pull request by ChrysmOre**
### What

Fixed the download URLs to have the correct format for Mongo document

### How to review

check code makes sense, tests pass

### Who can review

Anyone

### CODE REVIEW

File: handler/cantabular_metadata_export.go [hunk 1]
```diff
@@ -152,14 +152,14 @@ func (h *CantabularMetadataExport) exportTXTFile(ctx context.Context, e *event.C
...
-	download.URL = url
+	download.URL = h.generateDownloadURL(e, "txt")
 
 	return download, nil
 }
 
 func (h *CantabularMetadataExport) exportCSVW(ctx context.Context, e *event.CSVCreated, m dataset.Metadata, isPublished bool) (*dataset.Download, error) {
 	filename := h.generateCSVWFilename(e)
-	downloadURL := h.generateDownloadURL(e)
+	downloadURL := h.generateDownloadURL(e, "csv-metadata.json")
```
Comment (redhug1) [inline handler/cantabular_metadata_export.go:162]: Are you sure thats the correct extension for a csvw file ??????????????

Comment (ChrysmOre) [reply handler/cantabular_metadata_export.go:162]: Yes, 100%

Comment (ChrysmOre) [reply handler/cantabular_metadata_export.go:162]: https://github.com/ONSdigital/dp-download-service/blob/develop/service/service.go line 161 you can see the path. Yes, it seems completely daft. No, I have no idea why it is this.

Comment (DavidSubiros) [reply handler/cantabular_metadata_export.go:162]: Yes, the download service has some 'funny' things. I think that's why they want to refurbish it.

**Review by DavidSubiros — APPROVED** [review]
LGTM and @redhug1 comment has been responded

File: handler/cantabular_metadata_export.go (hunk 1 above)
Comment (redhug1) [reply handler/cantabular_metadata_export.go:162]: Hmmm. OK ... thats what it is then.

**Pull request merged**